src/app/components/cells/currency/currencyHelper.test.js

```diff
@@ -1,3 +1,4 @@
+import { describe, expect, it } from "vitest";
```

Comment:
Diese Changes könntest du rausnehmen, indem du in der vitest.config.test `test.globals` auf `true` setzt ( [Doku hier](https://vitest.dev/config/#globals) ).

Reply:
Ach cool. Danke 👍